src/commentbot/verify_projectfile.jl

```diff
@@ -47,18 +47,22 @@ function pfile_hasfields(p::Pkg.Types.Project)
     return true, nothing
 end
 
-function verify_projectfile_from_sha(reponame, sha; auth=GitHub.AnonymousAuth())
+function verify_projectfile_from_sha(reponame, commit_sha; auth=GitHub.AnonymousAuth(), subdir = "")
     project = nothing
     projectfile_found = false
     projectfile_valid = false
     err = nothing
     @debug("Getting gitcommit object for sha")
-    gcom = gitcommit(reponame, GitCommit(Dict("sha"=>sha)); auth=auth)
+    gcom = gitcommit(reponame, GitCommit(Dict("sha"=>commit_sha)); auth=auth)
     @debug("Getting tree object for sha")
-    t = tree(reponame, Tree(gcom.tree); auth=auth)
+    t = tree(reponame, Tree(gcom.tree); auth=auth, params = Dict(:recursive => true))
```

Comment:
Would it be worth only activating `recursive` in the subdir case? Otherwise there is no need to transfer all the deeper data.

Reply:
yeah, probably good to limit the data transfer when it's not needed. Fixed in https://github.com/JuliaRegistries/Registrator.jl/pull/287/commits/06747a2a3625b13cd64caa02de13d0a11f73c722